server/metrics/metrics.go

```diff
@@ -132,6 +132,9 @@ const (
 
 	/// Whether the request was allowed by quota manager.
 	QuotaAllowed = "quota_allowed"
+
+	// The type of request a not found error was initiated from
+	NotFoundErrorType = "type"
```

Comment:
We should call this `CacheRequestType` or something like that, no need to tie the label name to a specific metric.

Reply:
Done